toolchain/parse/handle_statement.cpp

```diff
@@ -179,8 +180,15 @@ auto HandleStatementReturn(Context& context) -> void {
   context.PushState(state);
 
   context.AddLeafNode(NodeKind::ReturnStatementStart, context.Consume());
-  if (!context.PositionIs(Lex::TokenKind::Semi)) {
+
+  if (auto var_token = context.ConsumeIf(Lex::TokenKind::Var)) {
```

Comment:
In state.def, please update StatementReturn comments according to this change.

Reply:
Done.